app/assets/javascripts/map/views/layers/Forma250CoverLayer.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * The Forma Coverage layer module for use on canvas.
+ *
+ * @return Forma250CoverLayer class (extends CanvasLayerClass)
+ */
+define([
+  'abstract/layer/CartoDBLayerClass',
+], function(CartoDBLayerClass) {
+
+  'use strict';
+
+  var Forma250CoverLayer = CartoDBLayerClass.extend({
+
+    options: {
+      cartocss: '#layer { polygon-fill: #FF6699; polygon-opacity: 0.2; line-width: 0; }',
```

Comment:
Even if it's just a single three attributes, can we have it in a different / separated file?